src/snackbar.jsx

```diff
@@ -236,7 +234,7 @@ const Snackbar = React.createClass({
       actionColor,
     } = this.constructor.getRelevantContextKeys(this.state.muiTheme);
 
-    const isSmall = this.state.deviceSize === this.constructor.Sizes.SMALL;
+    const isSmall = window.innerWidth < 768;
```

Comment:
True, on one hand I figured this shouldn't matter to much because:

1. Snackbars are generally temporary
2.  The resolution of small devices typically does not change

Though on second thought, there are some other considerations like the width as a result from a orientation change.